src/pages/home/base/mobile-core/release-notes.md

```diff
@@ -30,6 +36,10 @@ keywords:
 
 Downgrading from this version of Core is unsupported, as it will cause undefined behavior.
 
+<InlineAlert variant="warning" slots="text"/>
+
+Please do not use this release for tvOS. Instead use 4.2.1 release.
```

Comment:
I'd go with "Use the 4.2.1 release instead." rather than "Instead use...", since (to me at least), it reads better.